### ISSUE TITLE
Extend the template for Cython

### ISSUE THREAD

**Issue by mabl** [jacebrowning/template-python#45]
I've just started a new project and thought about using template-python. Unfortunately, I'll have to use Cython to speed up parts of my code. In the context of this framework, how do I proceed best in including Cython code?

I've found some resources on the web, which document my confusion perfectly:
http://pages.uoregon.edu/cfulton/posts/building_python_modules.html
http://comments.gmane.org/gmane.comp.python.cython.user/11186

Basically, I looks like the makefile could take over the generation of the .c files from the .pyx files and only the .c files are then mentioned as ext_modules in setuptools.setup.

Do you have any opinions or recommendations on this matter?

**Comment by jacebrowning** [comment]
Closing due to inactivity.
